Sources/CNIOLinux/shim.c

```diff
@@ -150,4 +150,21 @@ size_t CNIOLinux_CMSG_SPACE(size_t payloadSizeBytes) {
 
 const int CNIOLinux_SO_TIMESTAMP = SO_TIMESTAMP;
 const int CNIOLinux_SO_RCVTIMEO = SO_RCVTIMEO;
+
+int CNIOLinux_supports_udp_segment() {
+    #ifndef UDP_SEGMENT
```

Comment:
Also, why are we compile guarding for the absence of this flag here, where we rely on it being present in `ChannelOption.swift`?

Reply:
Yes, Linux has a stable ABI so the value of this flag is hardcoded and cannot change. We've done this with other socket options.